demos/textinput/index.php

```diff
@@ -32,355 +32,367 @@
-             <input type="text" data-clear-btn="true" name="text-7" id="text-7" value="" placeholder="Placeholder">
+				 <label for="text-7">Text input: data-clear-btn="true"</label>
+				 <input type="text" data-clear-btn="true" name="text-7" id="text-7" value="" placeholder="Placeholder">
 
-             <label for="search-5">Search:</label>
-             <input type="search" name="search-5" id="search-5" value="" placeholder="Placeholder">
+				 <label for="search-5">Search:</label>
+				 <input type="search" name="search-5" id="search-5" value="" placeholder="Placeholder">
 
-            <label for="textarea-6">Textarea:</label>
-            <textarea cols="40" rows="8" name="textarea-6" id="textarea-6" placeholder="Placeholder"></textarea>
-        </form>
+				<label for="textarea-6">Textarea:</label>
+				<textarea cols="40" rows="8" name="textarea-6" id="textarea-6" placeholder="Placeholder"></textarea>
+			</form>
         </div><!-- /demo-html -->
 
         <h2>Value/Text</h2>
 
         <div data-demo-html="true">
-        <form>
-             <label for="text-18">Text input:</label>
-             <input type="text" name="text-18" id="text-18" value="Value">
+			<form>
+				 <label for="text-18">Text input:</label>
+				 <input type="text" name="text-18" id="text-18" value="Value">
 
-             <label for="text-19">Text input: data-clear-btn="true"</label>
-             <input type="text" data-clear-btn="true" name="text-19" id="text-19" value="Value">
+				 <label for="text-19">Text input: data-clear-btn="true"</label>
+				 <input type="text" data-clear-btn="true" name="text-19" id="text-19" value="Value">
 
-             <label for="search-11">Search:</label>
-             <input type="search" name="search-11" id="search-11" value="Value">
+				 <label for="search-11">Search:</label>
+				 <input type="search" name="search-11" id="search-11" value="Value">
 
-            <label for="textarea-18">Textarea:</label>
-            <textarea cols="40" rows="8" name="textarea-18" id="textarea-18">Text</textarea>
-        </form>
+				<label for="textarea-18">Textarea:</label>
+				<textarea cols="40" rows="8" name="textarea-18" id="textarea-18">Text</textarea>
+			</form>
         </div><!-- /demo-html -->
 
+        <h2>Theming</h2>
+		
+        <div data-demo-html="true">
+			<form>
+				 <label for="search-10">Search:</label>
+				 <input type="search" name="search-10" id="search-10" value="" data-theme="b">
+			</form>
+        </div><!-- /demo-html -->
+		
+		<p>To have a light icon, remove the <code>ui-alt-icon</code> class by setting the widget's <code>classes</code> option with the <code>data-classes</code> attribute.</p>
+		
+        <div data-demo-html="true">
+			<form>
+				 <label for="search-12">Search:</label>
+				 <input type="search" name="search-12" id="search-12" value="" data-theme="b" data-classes="{ 'ui-textinput-search-icon': 'ui-icon ui-icon-search' }">
```

Comment:
Changing the search icon classes isn't working. The search icon only gets class ```ui-textinput-search-icon``` and the wrapper doesn't get the corner and shadow classes anymore.

See also https://github.com/jquery/jquery-mobile/issues/6757#issuecomment-127010902